tests/test_bridge_providers.py

```diff
@@ -1254,8 +1256,6 @@ def test_update_execution_status(
         expected_elapsed_time: int,
     ) -> None:
         """test_update_execution_status"""
-        monkeypatch.setattr("operate.ledger.DEFAULT_RPCS", TEST_RPCS)
```

Comment:
We were patching to use main RPCs. Now these will be defined in GH secrets and will be the normal RPCs - no need to patch here.